pdns/dnsdistdist/dnsdist-lua-bindings.cc

```diff
@@ -419,34 +420,34 @@ void setupLuaBindings(LuaContext& luaCtx, bool client, bool configCheck)
 
 #ifndef DISABLE_CLIENT_STATE_BINDINGS
   /* ClientState */
-  luaCtx.registerFunction<std::string (ClientState::*)() const>("toString", [](const ClientState& fe) {
+  luaCtx.registerFunction<std::string (ClientState::*)() const>("toString", [](const ClientState& frontend) {
     setLuaNoSideEffect();
-    return fe.local.toStringWithPort();
+    return frontend.local.toStringWithPort();
   });
-  luaCtx.registerFunction<std::string (ClientState::*)() const>("__tostring", [](const ClientState& fe) {
+  luaCtx.registerFunction<std::string (ClientState::*)() const>("__tostring", [](const ClientState& frontend) {
     setLuaNoSideEffect();
-    return fe.local.toStringWithPort();
+    return frontend.local.toStringWithPort();
   });
-  luaCtx.registerFunction<std::string (ClientState::*)() const>("getType", [](const ClientState& fe) {
+  luaCtx.registerFunction<std::string (ClientState::*)() const>("getType", [](const ClientState& frontend) {
     setLuaNoSideEffect();
-    return fe.getType();
+    return frontend.getType();
   });
-  luaCtx.registerFunction<std::string (ClientState::*)() const>("getConfiguredTLSProvider", [](const ClientState& fe) {
+  luaCtx.registerFunction<std::string (ClientState::*)() const>("getConfiguredTLSProvider", [](const ClientState& frontend) {
     setLuaNoSideEffect();
-    if (fe.tlsFrontend != nullptr) {
-      return fe.tlsFrontend->getRequestedProvider();
+    if (frontend.tlsFrontend != nullptr) {
+      return frontend.tlsFrontend->getRequestedProvider();
     }
-    else if (fe.dohFrontend != nullptr) {
+    if (frontend.dohFrontend != nullptr) {
       return std::string("openssl");
     }
     return std::string();
   });
-  luaCtx.registerFunction<std::string (ClientState::*)() const>("getEffectiveTLSProvider", [](const ClientState& fe) {
+  luaCtx.registerFunction<std::string (ClientState::*)() const>("getEffectiveTLSProvider", [](const ClientState& frontend) {
     setLuaNoSideEffect();
-    if (fe.tlsFrontend != nullptr) {
-      return fe.tlsFrontend->getEffectiveProvider();
+    if (frontend.tlsFrontend != nullptr) {
+      return frontend.tlsFrontend->getEffectiveProvider();
     }
-    else if (fe.dohFrontend != nullptr) {
+    if (frontend.dohFrontend != nullptr) {
```

Comment:
Or doqFrontend ?

Reply:
GH is having a hard time telling me exactly which line you are commenting on, but I think this is related to the TLS providers, ie `getConfiguredTLSProvider` or `getConfiguredTLSProvider`? For DoQ it's a bit tricky, I guess it should be `BoringSSL`?

Reply:
At least GH shows the correct line on my view. But yes, this is about `getConfiguredTLSProvider` and `getConfiguredTLSProvider`.
I agree.